plugins/ipqualityscore/plugin.spec.yaml

```diff
@@ -0,0 +1,784 @@
+        required: false
+        example: false
+      active_vpn:
+        title: Active VPN
+        description: Premium Account Feature - Identifies active VPN connections used by popular VPN services and private VPN servers.
+        type: boolean
+        required: false
+        example: false
+      active_tor:
+        title: Active TOR
+        description: Premium Account Feature - Identifies active TOR exits on the TOR network.
+        type: boolean
+        required: false
+        example: false
+      recent_abuse:
+        title: Recent Abuse
+        description: This value will indicate if there has been any recently verified abuse across our network for this IP address. Abuse could be a confirmed chargeback, compromised device, fake app install, or similar malicious behavior within the past few days.
+        type: boolean
+        required: false
+        example: false
+      bot_status:
+        title: Bot Status
+        description: Premium Account Feature - Indicates if bots or non-human traffic has recently used this IP address to engage in automated fraudulent behavior. Provides stronger confidence that the IP address is suspicious.
+        type: boolean
+        required: false
+        example: false
+      connection_type:
+        title: Connection Type
+        description: Classification of the IP address connection type as "Residential", "Corporate", "Education", "Mobile", or "Data Center".
+        type: string
+        required: false
+        example: 'Data Center'
+      abuse_velocity:
+        title: Abuse Velocity
+        description: Premium Account Feature - How frequently the IP address is engaging in abuse across the IPQS threat network. Values can be "high", "medium", "low", or "none". Can be used in combination with the Fraud Score to identify bad behavior.
+        type: string
+        required: false
+        example: 'none'
+      latitude:
+        title: Latitude
+        description: Latitude of IP address if available or "N/A" if unknown.
+        type: float
+        required: false
+        example: 34.0544014
+      longitude:
+        title: Longitude
+        description: Longitude of IP address if available or "N/A" if unknown.
+        type: float
+        required: false
+        example: -118.24410248
+  emailLookup:
+    title: Email Address reputation
+    description: Email Address reputation
+    input:
+      emailAddress:
+        title: Email Address
+        description: Email address for which information is desired
+        type: string
+        required: true
+        example: 'abc@gmail.com'
+      abuse_strictness:
+        title: Abuse Strictness
+        description: Set the strictness level for machine learning pattern recognition of abusive email addresses with the "recent_abuse" data point.
+        type: integer
+        required: true
+        default: 0
+        example: 0
+        enum:
+          - 0
+          - 1
+          - 2
+      fast:
```

Comment:
For input variables that do not have the required field. Could this please be added?